app/models/user.rb

```diff
@@ -3,4 +3,5 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+  has_many :restaurants
```

Comment:
Missing has_many reviews, dishes, ingredients, etc. (otherwise we can just leave out the relationship bw users and items).